convert-to-declarative-plugin/pom.xml

```diff
@@ -170,12 +166,6 @@
       <version>2.26</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-support</artifactId>
```

Comment:
It doesn't make sense to have test scope dependencies on plugins that are transitively compile scope dependencies. If the dependency was here to fix RequireUpperBoundsDeps issues, it would make more sense to move them to dependencyManagement, but it seems that things are ok just removing the explicit dependency. Same for credentials and maybe some others.

Reply:
A unit test is now missing dependency :)  pipeline-build-step